app.js

```diff
@@ -1,33 +1,40 @@
 // built-in package
-const http = require("http");
 
 // 3rd-party package
 require("dotenv").config();
 const express = require("express");
 const cors = require("cors");
 const morgan = require("morgan");
+
 // custom package
+const route = require("./api/routes");
+const dataSource = require("./api/models/dataSource");
+const { globalErrorHandler } = require("./api/utils/error");
+
+dataSource
+  .initialize()
+  .then(() => {
+    console.log("Data Source has been initialized!");
+  })
+  .catch((error) => {
+    console.error("Error during Data Source initialization", error);
+  });
 
 const app = express();
 
 app.use(cors());
 app.use(morgan("combined"));
 app.use(express.json());
+app.use(route);
+app.use(globalErrorHandler);
```

Comment:
```suggestion
app.use(route)

app.all('*', (req, res, next) => {
    const err = new Error(`Can't find ${req.originalUrl} on this server!`);
    err.statusCode = 404;
    next(err);
  });

app.use(globalErrorHandler)
```
routes에 작성되지 않은, 요청에 대해 찾을 수 없는 엔드포인트에 대하여 404 status code를 뱉을 수 있는 코드도 추가되면 좋을 것 같습니다.